src/pages/solution/adobe-campaign-classic/tabs/push-payloads.md

```diff
@@ -0,0 +1,161 @@
+---
+noIndex: true
+---
+
+<Variant platform="fcm" template="basic" repeat="9"/>
+
+### Configuration
+
+If using "remind later" functionality in a basic notification, it is required that you declare use of the `USE_EXACT_ALARM` permission in your `AndroidManifest.xml`:
+
+```xml
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.mycompany.myapp">
+    ...
+    <uses-permission android:name="android.permission.USE_EXACT_ALARM" />
+    ...
+</manifest>
+```
+
+### Properties
+
+The properties below are used to define the payload sent to FCM:
+
+| **Field** | **Required** | **Key** | **Type** | **Description** |
+| :-------- | :----------- | :------ | :------- | :-------------- |
+| Payload Version | ✅ | `adb_version` | string | Version of the payload assigned by the ACC authoring UI. |
+| Template Type | ✅ | `adb_template_type` | string | Informs the reader which properties may exist in the template object.<br />Basic template uses a value of "basic". |
+| Title | ✅ | `adb_title` | string | Text shown in the notification's title. |
+| Color - Title | ⛔️ | `adb_clr_title` | string | Text color for `adb_title`.<br />Represented as six character hex, e.g. `00FF00` |
+| Body | ✅ | `adb_body` | string | Text shown in message body when notification is collapsed. |
+| Expanded Body | ⛔️ | `adb_body_ex` | string | Body of the message when the message is expanded. |
+| Color - Body | ⛔️ | `adb_clr_body` | string | Text color for `adb_body`, `adb_body_ex`.<br />Represented as six character hex, e.g. `00FF00` |
+| Sound | ⛔️ | `adb_sound` | string | Sound played when notification is delivered. |
+| Image | ⛔️ | `adb_image` | string | URI of an image to be shown when notification is expanded. |
+| Link URI | ⛔️ | `adb_uri` | string | URI to be handled when user clicks the notification. |
+| Link Type | ⛔️ | `adb_a_type` | string | Type of link represented in `adb_uri` - one of "WEBURL", "DEEPLINK", or "OPENAPP".<br />Required if `adb_uri` is specified. |
+| Button(s) | ⛔️ | `adb_act` | string | **The value is an encoded JSON string.**<br />One to three buttons to create for the notification. If using "remind later" functionality, only two buttons may be used in this field.<br />When decoded, the string contains an array of the following objects:<br /><ul><li>`label` - text shown on the button's label</li><li>`uri` - URI to be handled when user clicks the button</li><li>`type` - Type of link represented in `uri` - one of "WEBURL", "DEEPLINK", or "OPENAPP".</li></ul> |
```

Comment:
👍 thanks for adding info about the button limit if remind later functionality is needed.